### PR TITLE
Nov 9: ARGO-docs-reorg-and-styling

### PR DESCRIPTION
**Pull request by kcullion**
- Styling fixes for docusaurus upgrade (mostly spacing)
- Split up DNA-Seq Analysis page into an individual page for each workflow
- new placeholder page for open access filtering
- Sidebar changes to integrate the new subsection for DNA-seq analysis
- Sidebar styling to visually indicate the subsection in the sidebar
- Revised Overview page for analysis pipelines that introduces DNA-seq and each individual page
- other content fixes to fix urls to the dna-seq pages and some rewording for pre-release data section

After this ticket https://github.com/icgc-argo/argo-docs/issues/462 is finished and tested, we can release this to production

### CODE REVIEW

**Review by hknahal — DISMISSED** [review]

File: docs/analysis-workflows/dna-sanger-wgs-vc.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+---
+id: dna-sanger-wgs-vc
+title: Sanger WGS Variant Calling
+sidebar_label: Sanger WGS Variant Calling
+platform_key: DOCS_DNA_PIPELINE
+---
+
+Whole genome sequencing (WGS) aligned CRAM files are processed through the Sanger WGS Variant Calling Workflow as tumour/normal pairs. The ARGO DNA Seq pipeline has adopted the [Sanger Whole Genome Sequencing Analysis Docker Image](https://quay.io/wtsicgp/dockstore-cgpwgs:2.1.0) as the base workflow. For details, please see the latest version of the [ARGO Sanger WGS Variant Calling workflow](https://github.com/icgc-argo/sanger-wgs-variant-calling/releases).
```
Comment (lindaxiang) [inline docs/analysis-workflows/dna-sanger-wgs-vc.md:8]: Can update to use new github repo: https://github.com/icgc-argo-workflows/sanger-wgs-variant-calling/releases

File: docs/analysis-workflows/dna-sanger-wxs-vc.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+id: dna-sanger-wxs-vc
+title: Sanger WXS Variant Calling
+sidebar_label: Sanger WXS Variant Calling
+platform_key: DOCS_DNA_PIPELINE
+---
+
+Whole exome sequencing (WXS) aligned CRAM files are processed through the Sanger WXS Variant Calling Workflow as tumour/normal pairs. The ARGO DNA Seq pipeline has adopted the [Sanger Whole Exome Sequencing Analysis Docker Image](https://quay.io/wtsicgp/dockstore-cgpwxs:3.1.6) as the base workflow. For details, please see the latest version of the [ARGO Sanger WXS Variant Calling workflow](https://github.com/icgc-argo/sanger-wxs-variant-calling/releases).
```
Comment (lindaxiang) [inline docs/analysis-workflows/dna-sanger-wxs-vc.md:8]: New repo: https://github.com/icgc-argo-workflows/sanger-wxs-variant-calling/releases

File: docs/analysis-workflows/dna-alignment.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+## Outputs
+
+- [Aligned Reads](/docs/data/reads#aligned-reads) CRAM and [Aligned Reads Index](/docs/data/reads#aligned-reads-index)
+- QC metrics files
+  - [Alignment Metrics](/docs/data/qc-metrics#aligned-reads-qc) files
+  - [OxoG Metrics](/docs/data/qc-metrics#aligned-reads-qc) files
+  - [Duplicates Metrics](/docs/data/qc-metrics#aligned-reads-qc) files
+  - [Read Group Metrics](/docs/data/qc-metrics#sequencing-qc) files for each submitted read group
+
```
[the remark this answers is not in the document]
Comment (kcullion) [reply docs/analysis-workflows/dna-alignment.md:35]: yes great idea..I have added to all the pages. thanks!

File: docs/analysis-workflows/dna-alignment.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+id: dna-alignment
+title: Alignment
+sidebar_label: Alignment
+platform_key: DOCS_DNA_PIPELINE
+---
+
+The ARGO Data Platform accepts raw sequencing data in both FASTQ and BAM (aligned or unaligned) format. The first processing step in the DNA-Seq Pipeline is uniformly aligning samples to the GRCh38 reference genome. For details, please see the latest version of the [ARGO DNA Alignment](https://github.com/icgc-argo/dna-seq-processing-wfs/releases).
```
[the remark this answers is not in the document]
Comment (kcullion) [reply docs/analysis-workflows/dna-alignment.md:8]: updated!

File: docs/analysis-workflows/dna-gatk-mutect2-vc.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
+---
+id: dna-gatk-mutect2-vc
+title: GATK Mutect2 Variant Calling
+sidebar_label: GATK Mutect2 Variant Calling
+platform_key: DOCS_DNA_PIPELINE
+---
+
+Whole genome/exome sequencing (WGS/WXS) aligned CRAM files are processed through the GATK Mutect2 Variant Calling Workflow as tumour/normal pairs. The ARGO DNA Seq pipeline has adopted the [Genome Analysis Toolkit Docker Image](https://hub.docker.com/r/broadinstitute/gatk) developed at Broad Institute as the base workflow. For details, please see the latest version of the [ARGO GATK Mutect2 Variant Calling workflow](https://github.com/icgc-argo/gatk-mutect2-variant-calling/releases).
```
[the remark this answers is not in the document]
Comment (kcullion) [reply docs/analysis-workflows/dna-gatk-mutect2-vc.md:8]: updated

**Review by ciaranschutte — APPROVED** [review]

**Pull request merged**